administration/src/components/cards/CreateCardForm.tsx

```diff
@@ -42,7 +42,7 @@ const CreateCardForm = (props: Props) => {
           <InputGroup
             placeholder='Erika Mustermann'
             autoFocus
-            intent={cardBlueprint.isFullNameValid() ? undefined : Intent.DANGER}
+            intent={cardBlueprint.isFullNameValid() && cardBlueprint.hasValidSize() ? undefined : Intent.DANGER}
```

Comment:
Imo it's enough to highlight where the creator of the card has to make adjustments. Especially as this seems to be an infrequent (but still possible) situation.

Reply:
Agreed. Maybe you can add a comment above, something like:
"If the size of the card is too large, show a warning at the name field as it is the only dynamically sized field".